src/features/class-graphql.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * GraphQL class file
+ *
+ * @package wp-curate
+ */
+
+namespace Alley\WP\WP_Curate\Features;
+
+use Alley\WP\Types\Feature;
+use WPGraphQL\AppContext;
+use GraphQL\Type\Definition\ResolveInfo;
+use WPGraphQL\Data\Connection\PostObjectConnectionResolver;
+
+/**
+ * Add support for GraphQL, if WPGraphQL plugin exists.
+ */
+final class GraphQL implements Feature {
+	/**
+	 * Post types allowed in GraphQL.
+	 */
+	private $allowed_post_types;
+
+	/**
+	 * Get the GraphQL Type by Post Type.
+	 *
+	 * @param  string $post_type_string Name of the post type. Ex: 'post'.
+	 *
+	 * @return string GraphQL Type name. Ex: 'Post'.
+	 */
+	private function get_graphql_type_by_post_type( $post_type_string ): string {
```

Comment:
Add type to function parameter also:
```suggestion
	private function get_graphql_type_by_post_type( string $post_type_string ): string {
```